plugins/generator-1.17.1/forge-1.17.1/templates/elementinits/potions.java.ftl

```diff
@@ -0,0 +1,61 @@
+<#--
+ # MCreator (https://mcreator.net/)
+ # Copyright (C) 2020 Pylo and contributors

```

Comment:
The copyright is not the new one. You should fix it.

Reply:
Where can I find the new one? I took this from the other 1.17 templates

Reply:
1.17 templates also don't have the right one. You can take the new text for years in a class inside `net.mcreator.themes`.

Reply:
@Goldorion oops, you can open PR to fix my templates if you get some extra time :D